doc/bucket-types.md

```diff
@@ -0,0 +1,35 @@
+[NooBaa Operator](../README.md) /
+
+# Bucket Types in NooBaa
```

Comment:
I think that this is great contribution, and I think it can be even better in noobaa-core